packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.jsx

```diff
@@ -132,13 +132,15 @@ const ProofOfOwnershipForm = ({ cards, updateAccountStatus }) => {
                                 setDocumentUploadErrors(form_errors);
                                 form_ref.current.setErrors(form_errors);
                                 form_ref.current.validateForm();
+                                setFormState({ ...form_state, ...{ is_btn_loading: false } });
                             } else {
+                                setFormState({ ...form_state, ...{ is_btn_loading: false } });
                                 // eslint-disable-next-line no-console
                                 console.warn(response);
```

Comment:
i think we dont need to have this.

Reply:
Hello @ali-hosseini-fs. This is necessary, to stop the loading state of the submit button